VideoLocker/src/main/java/org/edx/mobile/module/db/DbStructure.java

```diff
@@ -8,9 +8,12 @@
 public final class DbStructure {
     
     public static final String NAME = "downloads.db";
-    //Updated to Version 4 to add flag to indicate that video is only available for web
-    //Updated to Version 5 to create a new table to record learning history for assessment
-    public static final int VERSION = 5;
+    /**
+     * Updated to Version 4 to add flag to indicate that video is only available for web
+     * Updated to Version 5 to create a new table to record learning history for assessment
+     * Updated to Version 6 to swap every occurrence of username field to its SHA1 counterpart
+     */
```

Comment:
I don't think this should be the official javadoc of the `VERSION` constant (the `/**` at the start makes it a javadoc); if it's to be in a javadoc then it should probably be in the documentation of the `DbHelper#onUpgrade()` method.